### PR TITLE
Update component ontology rdfs:labels

### PR DESCRIPTION
**Pull request by smrgeoinfo**
Pattern is 'SWEET Ontology' {high level group} {Detail topic} , from most general to specific in the labels, so they'll sort in listings. 
Only change in each file should be the rdfs:label on the root 'owl:Ontology' entity.
addresses #85

### CODE REVIEW

File: src/humanJurisdiction.ttl [hunk 1]
```diff
@@ -18,7 +18,7 @@
                                                           <http://sweetontology.net/relaSpace> ,
                                                           <http://sweetontology.net/repr> ,
                                                           <http://sweetontology.net/reprSpaceGeometry> ;
-                                              rdfs:label "SWEET Ontology" ;
+                                              rdfs:label "SWEET Ontology Human Jursdiction" ;
```
Comment (lewismc) [inline src/humanJurisdiction.ttl]: Check spelling of ```Jurisdiction```

File: src/phenAtmoLightning.ttl [hunk 2]
```diff
@@ -18,7 +18,7 @@
                                                           <http://sweetontology.net/phenWave> ,
                                                           <http://sweetontology.net/propCharge> ,
                                                           <http://sweetontology.net/rela> ;
-                                              rdfs:label "SWEET Ontology" ;
+                                              rdfs:label "SWEET Ontology Phenomena Atmosphere Lightening" ;
```
Comment (lewismc) [inline src/phenAtmoLightning.ttl]: Check spelling of ```Lightening```

File: src/sweetAll.ttl [hunk 3]
```diff
@@ -230,6 +230,9 @@
                                                  <http://sweetontology.net/stateTimeGeologic> ,
                                                  <http://sweetontology.net/stateVisibility> ;
                                      rdfs:label "SWEET Ontology" ;
+									 	rdfs:comment """The semantic web for Earth and environmental terminology (SWEET) is an investigation in improving discovery and use of Earth science data, through software understanding of the semantics of web resources.  SWEET is a collection of ontologies conceptualizing a knowledge space for Earth system science, represented using the web ontology language (OWL). It includes both orthogonal concepts (space, time, Earth realms, physical quantities, etc.) and integrative science knowledge concepts (phenomena, events, etc.).  The ontology is modularized into 224 component files, each representing a different namespace, with numerous dependencies implemented by OWL imports between the namespace.
```
Comment (lewismc) [inline src/sweetAll.ttl]: @smrgeoinfo I would be tempted not to specify the number of component files as what happens if we add one more, take one away or produce more alignments...
The rest of this description is excellent.

File: src/sweetAll.ttl [hunk 4]
```diff
@@ -230,6 +230,9 @@
...
+
+The original version was developed by Robert Raskin at JPL (see Raskin and  Pan, 2005:  https://www.sciencedirect.com/science/article/pii/S0098300405001020 ). Stewardship has been taken over by the Earth Science Information Partners Federation (ESIPFed), managed through a GitHub repository at https://github.com/ESIPFed/sweet.  See https://github.com/ESIPFed/sweet/blob/master/CONTRIBUTING.md for information about providing comments or contributing to development of SWEET."""@en ;
```
Comment (lewismc) [inline src/sweetAll.ttl]: This is excellent, thank you

File: src/sweetAll.ttl (hunk 3 above)
Comment (smrgeoinfo) [reply src/sweetAll.ttl]: This brings up some deep questions about versioning-- right now v3.2.0 has 225 files-- one master, and 224 components.  I assume part of the logic for the components is that they could be versioned separately by additions, updates, deletions, or refactoring between components.   I assume if some component versions we updated then a new master version would be required that imported a different set of component versions.   In this model, the number of components in v3.2.0 master will always be 224.

**Comment by smrgeoinfo** [comment]
@lewismc  -- fixes made to typos  (76bf600) and added comment about number of components (9c9777a). this addresses change requests

**Comment by lewismc** [comment]
@smrgeoinfo please merge into master, thanks

**Comment by smrgeoinfo** [comment]
Merge done, thanks

**Comment by lewismc** [comment]
Thanks please clean up the open issues if there are any.

**Pull request merged**